src/main/java/org/owasp/wrongsecrets/challenges/docker/Challenge29.java

```diff
@@ -0,0 +1,62 @@
+package org.owasp.wrongsecrets.challenges.docker;
+
+
+import org.bouncycastle.util.encoders.Base64;
+import org.owasp.wrongsecrets.RuntimeEnvironment;
+import org.owasp.wrongsecrets.ScoreCard;
+import org.owasp.wrongsecrets.challenges.Challenge;
+import org.owasp.wrongsecrets.challenges.ChallengeTechnology;
+import org.owasp.wrongsecrets.challenges.Spoiler;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+import static org.owasp.wrongsecrets.RuntimeEnvironment.Environment.DOCKER;
+
+@Component
+@Order(29)
+public class Challenge29 extends Challenge {
+
+    private final String passcode = new String(Base64.decode("bW9pc2Rmbm93ZXkyMzRmaTMyaWNvOGFxdzQxMg=="));
```

Comment:
Let's do the obfuscation here a bit different as we discussed in Slack (RSA encryption of the passcode, with the RSA key file its location and the passphrase of the RSA key in the application.properties :)